build/docker/Dockerfile-bare

```diff
@@ -0,0 +1,43 @@
+# This dockerfile builds the zap stable release
+FROM ubuntu:16.04 as builder
```

Comment:
Alpine?

Reply:
Even when running multi-stage builds instead of having ubuntu, consider alpine:
```
FROM alpine as builder
RUN apk add --no-cache curl wget xmlstarlet unzip
```

This covers the build stage run commands used here just as well as ubuntu and pulls in 100MB less.

Reply:
You are totally right, I just assumed that it does not happen that much so I can just live with ubuntu but I can change it.